### ISSUE TITLE
data

### ISSUE THREAD

**Issue by zhy123321-cyber** [dengyecode/hourglassattention#13]
May I ask whether the data you input during training is damaged images or intact images? If it is damaged images, how can the loss be compared

**Comment by dengyecode** [comment]
The data loaded during the training phase consist of intact samples. The dataloader then introduces synthetic damage to these samples using the code in dataloader/data_loader.py.
